packages/lib/models/utils/types.ts

```diff
@@ -37,6 +37,7 @@ export interface LoadOptions {
 export interface FolderLoadOptions extends LoadOptions {
 	includeConflictFolder?: boolean;
 	includeTrash?: boolean;
+	excludeVirtualFolders?: boolean;
```

Comment:
Isn't it called "smartFilter"? What is the parent type when the trash is selected? If it's "smartFilter" I think that's how we should call it here